### PR TITLE
 chore: Add JavaDoc and fix some prefabs

### PR DESCRIPTION
**Pull request by skaldarnar**
doc: Add notes about event target and sender entity to ChatMessageEvent

fix: Remove components from prefabs that are no longer in engine
	- MinionMove is in Behaviors
	- Breather is in Breathing

### CODE REVIEW

File: engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java [hunk 1]
```diff
@@ -24,8 +24,9 @@
 import org.terasology.network.OwnerEvent;
 
 /**
- * A chat message
- *
+ * Indicate that a message should be printed to the chat console.
+ * <p>
+ * Needs to be send against a client entity, e.g., the targeted entity should have a {@link org.terasology.network.ClientComponent}.
```
Comment (jdrueckert) [inline engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java:29]:
```suggestion
 * Needs to be sent against a client entity, e.g., the targeted entity should have a {@link org.terasology.network.ClientComponent}.
```

File: engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java [hunk 2]
```diff
@@ -35,6 +36,17 @@
     protected ChatMessageEvent() {
     }
 
+    /**
+     * A chat message is associated with an entity that sent it.
```
Comment (jdrueckert) [inline engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java:40]:
```suggestion
     * A chat message is associated with the entity that sent it.
```

File: engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java [hunk 3]
```diff
@@ -35,6 +36,17 @@
...
+     * <p>
+     * This entity may often be a player, but it can be any entity with a {@link org.terasology.logic.common.DisplayNameComponent}.
+     * If the entity also has a {@link org.terasology.network.ColorComponent} the sender's name will be colorized.
```
Comment (jdrueckert) [inline engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java:43]:
```suggestion
     * If the entity also has a {@link org.terasology.network.ColorComponent}, the sender's name will be colorized.
```

Comment (jdrueckert) [reply engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java:43]: also not sure whether it's "colorized" or "coloured"

Comment (skaldarnar) [reply engine/src/main/java/org/terasology/logic/chat/ChatMessageEvent.java:43]: 🤷‍♂️ 🎨

**Pull request merged**